packages/web-pkg/src/components/FilesList/ResourceLink.vue

```diff
@@ -3,33 +3,43 @@
     :is="componentType"
     v-bind="componentProps"
     v-if="isResourceClickable"
-    :target="linkTargetBlank"
-    :aria-describedby="opensInNewWindowDescriptionId"
+    :target="linkTarget"
     :draggable="false"
     @dragstart.prevent.stop
     @click="emitClick"
   >
-    <slot :opens-in-new-window-description-id="opensInNewWindowDescriptionId" />
+    <slot />
   </component>
   <span v-else>
     <slot />
   </span>
 </template>
 
 <script lang="ts">
-import uniqueId from 'design-system/src/utils/uniqueId'
+import { useConfigStore } from '../../composables'
+import { storeToRefs } from 'pinia'
+import { computed, PropType, unref } from 'vue'
+import { RouteLocationRaw } from 'vue-router'
 
 /**
  * Wraps content in a resource link
  */
 export default {
   name: 'ResourceLink',
   props: {
+    /**
+     * The resource folder link
+     */
+    link: {
+      type: Object as PropType<RouteLocationRaw>,
+      required: false,
+      default: null
+    },
     /**
      * The resource folder link
      */
     folderLink: {
-      type: Object,
+      type: Object as PropType<RouteLocationRaw>,
       required: false,
       default: null
     },
```

Comment:
Now there is `link` and `folderLink`, I don't understand the difference 🤔 Do we need both, can't we use one for all? I would at least improve on the variable naming and the doc block above.

Reply:
We might be able to, we need to check